### DISCUSSION TITLE
Nozzle doesn't touch the bed after Z-home

### DISCUSSION THREAD

**Discussion by lgmenezes87** [Jyers/Marlin#886] [Q&A] [answered]
Sorry - I was making a dumb mistake :)

**Comment by Jyers** [comment, marked as answer]
Have you checked that your 2.01 z offset is actually correct? Although offsets should be consistent between firmwares, it could simply be the case that you need to adjust it.

**Reply by lgmenezes87** [reply]
I'm sorry, it was set to 2.01 instead of -2.01.
Feel free to close thread!
